e2e_tests/tests/config.py

```diff
@@ -176,6 +180,10 @@ def set_pt_image(config: Dict[Any, Any]) -> Dict[Any, Any]:
     return set_image(config, PT_CPU_IMAGE, PT_GPU_IMAGE)
 
 
+def set_pt2_image(config: Dict[Any, Any]) -> Dict[Any, Any]:
```

Comment:
don't see this method being called anywhere?

Reply:
Oops, nice catch. Meant to add it to a nightly test.